Makefile

```diff
@@ -1,13 +1,20 @@
 ARCHS=linux linux64 mac64 win32
-MANIFEST=extension/manifest.json
 
-VERSION=`grep -Eo '"version"\:\s*"[^"]+' $(MANIFEST) | grep -Eo '[0-9].*'`
+ROOT_PATH=/pub/labs/fxos-simulator/adb-helper/
```

Comment:
I've been wondering we need a new place for the new extension.  Now we have a new good name, devtools-adb-extension, we should use this.
 /pub/devtools/devtools-adb-extension/
?
I am not sure we can create the devtools directory on the /pub.